### DISCUSSION TITLE
Merging Objects and Clustering

### DISCUSSION THREAD

**Discussion by shanisa11** [satijalab/seurat#5411] [Q&A]
Hi,

I am new to scRNA-seq analysis and wanted to ask a question regarding object merging. 
If I have two conditions, 4 different libraries per each condition, and I have created 8 objects (one per each library), one object for condition 1 merged,  one object for condition two merged, and one object for both conditions merged containing all 8 libraries- (total 11 objects), 
on which should I preform the guided clustering tutorial? Should I do it for each separate library, or only for the merged total object?

**Comment by samuel-marsh** [comment]
Hi,

So unfortunately there is no right answer necessary to this and depends on your project goals and analysis techniques and individual experiment.  I would say the analyzing each library individually is probably not what you want to do.  But depending on your goals and experiment it may be better to perform analysis on each condition separately and then analyze together or may be better to analyze all together.  Or maybe some combination of both.  It's something that you will have to determine based on experiment goals and results of each analysis.

Best,
Sam
